src/materials/ConcreteThermalMoisture.C

```diff
@@ -19,6 +19,10 @@
 #include "libmesh/quadrature.h"
 
 registerMooseObject("BlackBearApp", ConcreteThermalMoisture);
+registerMooseObjectRenamed("BlackBearApp",
+                           PorousMediaBase,
+                           "07/31/2021 00:01",
+                           ConcreteThermalMoisture);
 
```

Comment:
This was removed in an earlier PR. Please update your branch.

Reply:
corrected now.